### PR TITLE
Feature/image multi text block

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

File: src/blocks/ImageMultiTextBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  <SectionWrapper color={color && color.color} size="sm">
+    <ContentWrapper>
+      <Wrapper imagePosition={image_position}>
+        <Col pad={image_position === 'left' ? 'right' : 'left'}>
+          <img src={image} />
+        </Col>
+        <Col pad={image_position === 'left' ? 'left' : 'right'}>
+          {text_items.map((item) => (
+            <div key={item._uid}>
+              <h3>{item.title}</h3>
```
Comment (isaacSennerholt) [inline src/blocks/ImageMultiTextBlock.tsx]: h3 is a suitable header type, though it seems to be larger in font-size here than in the Figma file. Now it's ~31.984px (base 16px) while in the Figma it's 20px.

Comment (palmfjord) [reply src/blocks/ImageMultiTextBlock.tsx]: woops, nice catch!

File: src/blocks/ImageMultiTextBlock.tsx [hunk 2]
```diff
@@ -28,6 +28,16 @@ const Col = styled('div')(({ pad }: { pad: 'left' | 'right' }) => ({
   },
 }))
 
+const Title = styled('h3')({
+  fontSize: 20,
```
Comment (isaacSennerholt) [inline src/blocks/ImageMultiTextBlock.tsx]: rem?

Comment (palmfjord) [reply src/blocks/ImageMultiTextBlock.tsx]: Yes! I was lazy, no good 🙅‍♀️

**Pull request merged**